src/pykeen/regularizers.py

```diff
@@ -77,18 +77,26 @@ def forward(self, x: torch.FloatTensor) -> torch.FloatTensor:
         """Compute the regularization term for one tensor."""
         raise NotImplementedError
 
-    def update(self, *tensors: torch.FloatTensor) -> None:
+    def update(self, *tensors: torch.FloatTensor) -> bool:
         """Update the regularization term based on passed tensors."""
+        # TODO if not self.training or not torch.is_grad_enabled() or (self.apply_only_once and self.updated):
```

Comment:
@mberr we changed this part of the update step in the new code, should we also apply it to the old code? (the checking if in training mode and that grad is enabled)